src/qtiItem/helper/maxScore.js

```diff
@@ -822,5 +822,21 @@ export default {
             max = 0;
         }
         return max;
+    },
+    pciInteraction: function pciInteraction(interaction) {
```

Comment:
**chore:** PCI means `Portable Custom Interaction`, the name is a bit redundant. Moreover, it doesn't reflect well the intent.

**suggestion:** I suggest using the same convention as for other interactions, say `customInteractionBased`. In the end, it applies to custom interactions.
```suggestion
    customInteractionBased(interaction) {
```